cmd/build/helmify/static/README.md

```diff
@@ -166,6 +166,7 @@ _See [Exempting Namespaces](https://open-policy-agent.github.io/gatekeeper/websi
 | controllerManager.hostNetwork                 | Enables controllerManager to be deployed on hostNetwork                                                                                                                                                                                             | `false`                                                                   |
 | controllerManager.dnsPolicy                   | Set the dnsPolicy for controllerManager pods                                                                                                                                                                                                        | `ClusterFirst`                                                            |
 | controllerManager.securityContext             | Security context applied on the container                                                                                                                                                                                                           | `{ "allowPrivilegeEscalation": false, "capabilities": "drop": [all], "readOnlyRootFilesystem": true, "runAsGroup": 999, "runAsNonRoot": true, "runAsUser": 1000 }` |
+| controllerManager.tlsMinVersion               | Set the minumum supported TLS version for controllerManager                                                                                                                                                                                          | `1.3`                                                                    |
```

Comment:
```suggestion
| controllerManager.tlsMinVersion               | Set the minimum supported TLS version for validating and mutating webhook servers                                                                                                                                                                                          | `1.3`                                                                    |
```

Reply:
Fixed in https://github.com/open-policy-agent/gatekeeper/pull/2289/commits/f1362cecf04fd1f88069df3573d7128b58dea91e, thank you!